modin/core/storage_formats/pandas/utils.py

```diff
@@ -18,6 +18,24 @@
 import pandas
 
 
+class NullContextManager(object):  # noqa: PR01
+    """
+    Dummy context manager that tries to mimic `nullcontext` for older Python versions.
+
+    Python 3.6 currently does not support `nullcontext` which is used in parsers.py.
+    This class tries provide some backwards compatibility.
+    """
+
+    def __init__(self, resource=None):
+        self.resource = resource
+
+    def __enter__(self):  # noqa: GL08
+        return self.resource
+
+    def __exit__(self, *args):  # noqa: GL08
+        pass
```

Comment:
I think you can move this one and generalize, and it would take less code to do so:
https://github.com/modin-project/modin/blob/23c40a666dd8af7f781f1c10e0efea15f562d72a/modin/pandas/test/test_general.py#L32-L35

Reply:
Yes this is much nicer to use.